### DISCUSSION TITLE
issue: Focus on error works differently in Firefox

### DISCUSSION THREAD

**Discussion by lahdeero** [react-hook-form/react-hook-form#10123] [Q&A]
### Version Number

7.43.1

### Codesandbox/Expo snack

https://codesandbox.io/s/smoosh-tree-g5nfln

### Steps to reproduce

1. Go to bottom of the page
2. Click on Submit
3. See input field and error in Chrome (and Safari) but not with Firefox

### Expected behaviour

Scroll to input with error should work the same way with Firefox

### What browsers are you seeing the problem on?

Firefox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
This is not related to this library.
